src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs

```diff
@@ -127,6 +134,12 @@ public override void OnRename(Series series)
         public override ValidationResult Test()
         {
             var failures = new List<ValidationFailure>();
+            var scriptsPath = GetScriptsFolder();
```

Comment:
Since the user can change the scripts folder, wouldn't this make it possible for an attacker to change the scripts folder and still execute whatever they like? (`../../../usr/bin`)

Reply:
That is a concern (but also allowed on other apps), that'd be my argument for removing `Arguments` (no pun intended).